packages/cozy-client/src/registry.js

```diff
@@ -0,0 +1,55 @@
+import 'url-search-params-polyfill'
+import termUtils from './terms'
+import { APP_TYPE } from './constants'
+
+const queryPartFromOptions = options => {
+  const query = new URLSearchParams(options).toString()
+  return query ? `?${query}` : ''
+}
+
+class Registry {
+  constructor(options) {
+    if (!options.client) {
+      throw new Error('Need to pass a client to instantiate a Registry API.')
+    }
+    this.client = options.client
+  }
+  /**
+   * Installs or updates an app from a source.
+   *
+   * Accepts the terms if the app has them.
+
+   * @param  {RegistryApp} app - App to be installed 
+   * @param  {string} source - String (ex: registry://drive/stable)
+   * @return {Promise}
+   */
+  async installApp(app, source) {
```

Comment:
About this API, I am just thinking that it can't be used by any other apps except the Store since the Stack hardcoded the slug authorized to install applications

Reply:
Asked the back for a solution ;)